### ISSUE TITLE
POS Duplicate Invoice 

### ISSUE THREAD

**Issue by Olawale1** [frappe/erpnext#11000]
Hi,

I just noticed a Sales Invoice got created twice after syncing from POS. The good thing is that it was flagged as Row 1 & 2 so I spotted it easily but wondering why this happened at all in the first place?

![pos duplicate invoice issue 1](https://user-images.githubusercontent.com/9269337/31047176-9ba52e7c-a5fd-11e7-9064-2808fb6daae8.png)

Thanks

**Comment by rohitwaghchaure** [comment]
Hi @Olawale1 

How did you sync the offline invoices in the local storage by clicking on button Sync Offline Invoices or automatic by the system?

**Comment by Olawale1** [comment]
Hi @rohitwaghchaure 

Trust you're doing great. It was by clicking on the 'Sync Offline Invoices' button. Actually, there was another issue which stopped Invoices from syncing earlier. The cashier filled 'Address Line 1' when creating a new customer but didn't fill anything in the City and State fields. This then caused an error that stopped the syncing

I had to amend the json in local storage and then I used the 'Sync Offline Invoices' button

Thanks
